ibm/service/vpc/resource_ibm_is_share_accessor_binding_operations.go

```diff
@@ -0,0 +1,123 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
```

Comment:
missing test files for this

Reply:
modified resource to ibm_is_share_delete_accessor_binding
added test file